appveyor.yml

```diff
@@ -0,0 +1,32 @@
+version: "{build}"
```

Comment:
Changed...just because I like you

On Thu Jan 22 2015 at 12:53:06 PM Bryan McLellan <notifications@github.com>
wrote:

> In appveyor.yml
> <https://github.com/chef/ohai/pull/465#discussion_r23408763>:
>
> > @@ -0,0 +1,32 @@
> > +version: "{build}"
>
> seemed useful in concept? *shrug*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/ohai/pull/465/files#r23408763>.
>